src/googletest.h

```diff
@@ -536,6 +546,14 @@ static inline bool MungeAndDiffTestStderr(const string& golden_filename) {
   return true;
 }
 
+static inline bool MungeAndDiffTestStderr(const string& golden_filename) {
+  return MungeAndDiffTestStderrOrStdout(golden_filename, false);
```

Comment:
`bool` parameters are hard to follow. It would be clearer if you pass `CapturedStream* cap` or an index into `s_captured_streams` to avoid code repetition.